### PR TITLE
Automatically load workflows when switching accounts

### LINKED ISSUE #563: Figure out how to load workflows on AWS
It's helpful for demo and QA purposes.

Loading workflows is currently done via rake task, not in the UI.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #563

### CODE REVIEW

File: app/models/account.rb [hunk 1]
```diff
@@ -46,10 +46,18 @@ def self.canonical_cname(cname)
...
   def switch!
     solr_endpoint.switch! if solr_endpoint
     fcrepo_endpoint.switch! if fcrepo_endpoint
     redis_endpoint.switch! if redis_endpoint
+    load_workflows
+  end
+
+  # Ensure that some workflows have been loaded into the database
+  def load_workflows
+    return if Sipity::Workflow.any?
```
Comment (mjgiarlo) [inline app/models/account.rb]: @jcoyne Why do we need this check? If I have, say, 2 workflows available and I run the importer *n* times, I will still wind up with 2 `Sipity::Workflow`s. Performance related, I guess?

**Comment by mjgiarlo** [comment]
@jcoyne Left one question. Otherwise, happy to merge.

File: app/models/account.rb (hunk 1 above)
Comment (jcoyne) [inline app/models/account.rb]: Yes. The importer reimports the workflows when called, so it's very slow.

Comment (mjgiarlo) [inline app/models/account.rb]: :ok_hand:

**Pull request merged**